react/components/SearchResult.js

```diff
@@ -234,7 +235,7 @@ class SearchResult extends Component {
                 <ExtensionPoint id="not-found" />
               </div>
             )}
-            {pagination === PAGINATION_TYPE.SHOW_MORE ? (
+            {pagination === PAGINATION_TYPE.SHOW_MORE || infiniteScrollError ? (
```

Comment:
Can this error go away with a retry?

Reply:
Yes! The reason for this line is because when there is an error, you can't trigger the infinite scrolling again, so it gets impossible to call `fetchMoreNext` once an error appears with infinite loading. This line makes sure a button appears on these cases and then you go back to infinite scrolling once you have fetched more results successfully since this resets the infinite scrolling behavior!